docs/API.md

```diff
@@ -210,7 +210,7 @@ $ curl -X POST --header "Content-Type:application/json" \
 
 > **Note:** This route will remove and recreate all service containers, as volumes can only be removed when their associated containers are removed. On devices with supervisor version lower than 7.22.0, replace all `BALENA_` variables with `RESIN_`, e.g. `RESIN_SUPERVISOR_ADDRESS` instead of `BALENA_SUPERVISOR_ADDRESS`.
 
-Clears the user application's `/data` folder.
+Clears the device's `/data` folder and named volumes.
```

Comment:
I'm not sure about this change, it might require some archeology, but I think for single container, the supervisor used a `/data/` folder for the app instead of a volume.

Reply:
@pipex I changed this to something else. how about now?